tools/ci/crate-range.py

```diff
@@ -179,12 +178,18 @@ def main():
 
     subparser_generate_matrix = subparsers.add_parser("generate-matrix", help="Generate a test matrix")
     subparser_generate_matrix.add_argument("-b", type=int, dest="batches", required=True, help="Number of batches")
+    subparser_generate_matrix.add_argument("-f", required=True, type=str, dest="folder",
```

Comment:
`-f` made me think this was `--force`, maybe we should use a long flag for this?